tools/pipelines/templates/include-vars.yml

```diff
@@ -101,3 +101,14 @@ variables:
 - ${{ if ne(parameters.releaseBuildOverride, 'none') }}:
   - name: release
     value: ${{ parameters.releaseBuildOverride }}
+
+- ${{ if eq(parameters.packageTypesOverride, 'none') }}:
+  - ${{ if eq(variables.shouldPublish, true) }}:
+    - name: releaseAlphaBetaBuild
+      value: $[variables.releaseBuild]
+  - ${{ if ne(variables.shouldPublish, true) }}:
+    - name: releaseAlphaBetaBuild
+      value: none
+- ${{ if ne(parameters.packageTypesOverride, 'none') }}:
+  - name: releaseAlphaBetaBuild
+    value: ${{ parameters.packageTypesOverride }}
```

Comment:
Actually, I don't think this needs to be defined here at all. Putting variables here is sort of making them "global" to the whole pipeline. But this is only needed in the build-npm-package workflow. So instead of declaring a variable here, revert these changes and add a parameter to the build-npm workflow, something like this:

```yaml
# The TypeScript types that should be published with the package. Note that setting this to a value other than 'none' will cause the version of the package to change as well.
- name: packageTypes
  type: string
  default: "none"
```